ibc-core/ics02-client/types/src/height.rs

```diff
@@ -89,6 +89,16 @@ impl Height {
     pub fn decrement(&self) -> Result<Height, ClientError> {
         self.sub(1)
     }
+
+    pub fn since(&self, other: Height) -> Option<i64> {
+        if self.revision_number != other.revision_number
+            || self.revision_height < other.revision_height
+        {
+            return None;
+        }
+
+        Some((self.revision_height - other.revision_height) as i64)
+    }
```

Comment:
@Farhad-Shabani let me know if introducing this method here is alright. otherwise, I will add it separately in the testkit.

Reply:
Let's keep this within the `ibc-testkit`, especially since it's used to acquire the latest timestamp. I'm skeptical about the necessity for it after refactoring the mocks.

Reply:
Let's keep this within the `ibc-testkit`, especially since it is solely used to obtain the latest timestamp. I'm skeptical about the necessity for it after refactoring the mocks.

Reply:
No worries. I was thinking the same. I used a local method [a3396cc](https://github.com/cosmos/ibc-rs/pull/1051/commits/a3396cc54d95b2b1cf86a3b512a8994a8eb7a9c7) and removed `Height::since` from ics02 types [12dd27b](https://github.com/cosmos/ibc-rs/pull/1051/commits/12dd27b73f0e1b349d5cf30917bf34b31bf796a8)